lib/sqs.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+const assert = require('assert');
+const chunk = require('lodash.chunk');
+const throat = require('throat');
+
+const { ajv, validate } = require('./ajv');
+const AwsDriverError = require('./aws-driver-error');
+const Driver = require('./driver');
+
+const BATCH_SIZE = 10;
+const DATA_TYPE_REGEX = new RegExp('^(String)((\\.)(\\w+))?$');
+const RETRY_COUNT = 2;
+
+const MESSAGE_ATTRIBUTE_SCHEMA = {
+  type: 'object',
+  required: [
+    'dataType',
+    'stringValue'
+  ],
+  properties: {
+    dataType: {
+      type: 'string',
+      pattern: DATA_TYPE_REGEX.source
+    },
+    stringValue: { type: 'string' }
+  },
+  additionalProperties: true
+};
+
+const SQS_MESSAGE_SCHEMA = {
```

Comment:
Technically, this does not support SQS message (with capitalized properties). May fix this when I get spare cycles.